### DISCUSSION TITLE
求教Mac如何开启Browser Dialer

### DISCUSSION THREAD

**Discussion by ironman2047** [XTLS/Xray-core#831] [Q&A]
Qv2ray内核使用xray-core，以下是我尝试的步骤：

终端执行命令：
$ vi ~/.bash_profile

加入并保存：
export XRAY_BROWSER_DIALER=http://127.0.0.1:54321
（也尝试过加入：export XRAY_BROWSER_DIALER=127.0.0.1:54321）

终端执行命令：
$ source ~/.bash_profile

使用浏览器打开页面
http://127.0.0.1:54321

结果：
转发页面无法打开，"This site can’t be reached"

**Comment by ghost** [comment]
不要再用qv2ray了，已停止更新
